ceph-dev-new-build/build/validate_mingw

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -ex
+
+# At the moment, we only support cross compiling Windows binaries using
+# Ubuntu or SUSE distros.
+( source /etc/os-release
+  case $ID in
+    opensuse*|suse|sles)
+      exit 0
+      ;;
+    ubuntu)
+      exit 0
+      ;;
+    *)
+      exit 1
+      ;;
+  esac) || exit 0
+
+if [ "${DIST}" != "windows" ]; then
+  exit 0
```

Comment:
I think we're supposed to return 1 for other DIST values. I can update those checks but I'll wait for confirmation.

Reply:
If you exit w/ `1`, it will fail the jenkins builder when in actuality all we really want to do is stop the processing of the shell script (where the validate/build/etc) scripts are merged together and run as a single unit.

Reply:
Yeah, actually, this is correct.  So maybe my [approach](https://github.com/ceph/ceph-build/pull/1722#discussion_r582896553) is also wrong still.

Reply:
Right, the `validate_XYZ` scripts should exit (cleanly) if we are on the wrong DEB vs RPM but then also exit cleanly if we don't match on the `windows` check.